raiden/network/transport/matrix/rtc/aio_queue.py

```diff
@@ -17,46 +18,33 @@ def make_wrapped_greenlet(target: Callable, *args: Any, **kwargs: Any) -> Future
     return wrapped_glet
 
 
-class AGTransceiver:
-    def __init__(self) -> None:
-        self.peer_connections: Dict[Address, RTCPeerConnection] = dict()
-        self.event_to_aio_queue = AGQueue()
-        self.event_to_gevent_queue = AGQueue()
-
-    def send_event_to_aio(self, event: Dict[str, Any]) -> None:
-        self.event_to_aio_queue.put(event)
-
-    async def aget_event(self, timeout):
-        try:
-            event = await self.event_to_aio_queue.aget(timeout)
-        except Empty:
-            return {"type": "timeout"}
-        return event
-
-    async def send_event_to_gevent(self, event) -> None:
-        await self.event_to_gevent_queue.aput(event)
+@dataclass
+class RTCPartner:
+    partner_address: Address
+    pc: RTCPeerConnection
+    channel: Optional[RTCDataChannel] = None
 
-
-class AGQueue(Queue):
-    async def aget(self, timeout):
-        return await make_wrapped_greenlet(self.get, timeout=timeout)
-
-    async def aput(self, item: Any) -> None:
-        await make_wrapped_greenlet(self.put, item)
+    def create_channel(self, node_address: Address) -> None:
+        lower_address = my_place_or_yours(node_address, self.partner_address)
+        higher_address = self.partner_address if lower_address == node_address else node_address
+        channel_name = (
+            f"{to_checksum_address(lower_address)}|{to_checksum_address(higher_address)}"
+        )
+        self.channel = self.pc.createDataChannel(channel_name)
 
 
 class AGLock:
```

Comment:
What does `AGLock` stand for?

Reply:
It's Aio Gevent Lock. A lock which can be used in both worlds. Currently it is not used, and I have not decided yet what to do with this code.